packages/stylelint-config/package.json

```diff
@@ -0,0 +1,52 @@
+{
+	"name": "@wordpress/stylelint-config",
+	"version": "17.0.0",
+	"description": "stylelint config for WordPress development.",
+	"author": "The WordPress Contributors",
+	"license": "MIT",
+	"keywords": [
+		"stylelint",
+		"stylelint-config",
+		"stylelint-config-wordpress",
+		"wordpress"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/stylelint-config/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git",
+		"directory": "packages/stylelint-config"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"files": [
+		"CHANGELOG.md",
+		"LICENSE",
+		"README.md",
+		"index.js",
+		"scss.js"
+	],
+	"main": "index.js",
+	"dependencies": {
+		"stylelint-config-recommended": "^3.0.0",
+		"stylelint-config-recommended-scss": "^4.2.0",
+		"stylelint-scss": "^3.17.2"
+	},
+	"peerDependencies": {
+		"stylelint": "^10.1.0 || ^11.0.0 || ^12.0.0 || ^13.0.0"
+	},
+	"npmpackagejsonlint": {
```

Comment:
Should we relax it for the Gutenberg repository or maybe even at `@eordpress/scripts` level?

I couldn't find at first how the linter passes and I tried to find out if there is a bug and I discovered this: https://github.com/tclindner/npm-package-json-lint/issues/221 😃

Reply:
Yes, [based on my comment above](https://github.com/WordPress/gutenberg/pull/22777#issuecomment-745200635) I want to change this. But this requires to also update the main package-lock.json to get the tests passing again and this may result in the deprecation warning visible until everything is switched to the new package. So I was planning to do this right after this PR gets merged. Not sure if there's a better way?

Reply:
I guess we could just go with `^13.0.0` for now.

Reply:
It's complicated. I'm fine with a follow-up PR if that helps :)